pkg/dockerregistry/testutil/fakeopenshift.go

```diff
@@ -39,6 +39,7 @@ func NewFakeOpenShift() *FakeOpenShift {
 // NewFakeOpenShiftWithClient constructs a fake client associated with
 // the stateful fake in-memory OpenShift reactors. The fake OpenShift is
 // available for direct interaction, so you can make buggy states.
+// TODO: remove the FakeOpenshift as the legacy client is not needed anymore
```

Comment:
@legionus @dmage i'm leaving this as a techdebt for you.

Reply:
@mfojtik I guess you are about `testclient.Fake`, not `FakeOpenShift`.

Reply:
testclient.Fake is not longer needed in registry, we should use imagefake.Clientset{} from generated client (similar to other clients required for openshift resources....).